src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -194,9 +198,40 @@ export const CustomerInstitutionMetadataForm = ({
               )}
             </Field>
 
+            <Field name={CustomerInstitutionFieldNames.RboInstitution}>
+              {({ field }: FieldProps<boolean>) => (
+                <div>
+                  <FormLabel
+                    component="legend"
+                    sx={{
+                      fontWeight: 'bold',
+                      marginTop: '-1rem',
```

Comment:
Negative verdier for margin og padding er ting man helst vil unngå, da det som regel er lagt in for å fjerne effekten av noe annet. Da er det bedre å fjerne rotårsaken, da det gir enklere kode heller enn å ha to motstridende effekter.
I tilfellet her kan du fjerne `marginTop`:
```suggestion
```
og wrappe dette i en ny div:
```
<div>
  <Field name={CustomerInstitutionFieldNames.NviInstitution}>
      (...)
   </Field>
   <Field name={CustomerInstitutionFieldNames.RboInstitution}>
      (...)
   </Field>
</div>
```
Da kan vi fjerne erstatte begge `<div>`  som ligger inn `<Field>` med `<>`  også